ignite/pkg/cosmosgen/templates/root/client.ts.tpl

```diff
@@ -0,0 +1,51 @@
+import { GeneratedType, OfflineSigner, EncodeObject, Registry } from "@cosmjs/proto-signing";
+import { StdFee } from "@cosmjs/launchpad";
+import { SigningStargateClient } from "@cosmjs/stargate";
+import { Env } from "./env";
+import { UnionToIntersection, Return, Constructor } from "./helpers";
+import { Module } from "./modules";
+
+const defaultFee = {
+  amount: [],
+  gas: "200000",
+};
+
+export class IgniteClient {
+	static plugins: Module[] = [];
+  env: Env;
+  signer: OfflineSigner;
+  registry: Array<[string, GeneratedType]>
```

Comment:
`Property 'registry' has no initializer and is not definitely assigned in the constructor.`

I'm trying to write a minimal client usage in order to integrate that in the integration tests, but I got this error when I try to compile `client.ts`. 

Sounds like I can fix that with 
```suggestion
  registry: Array<[string, GeneratedType]> = [];
```

If it's nothing, sorry in advance I know nothing about ts ^^' (I approved only the go part)